api-reference/_hidden/dxDataGridColumn/calculateGroupValue.md

```diff
@@ -4,16 +4,170 @@ type: String | function(rowData)
 ---
 ---
 ##### shortDescription
-Specifies a field name or a function that returns a field name or a value to be used for grouping column cells.
+A field name or a function that returns a field name or value by which to group cells. This value is also used to sort **DataGrid**'s groups.
 
 ##### param(rowData): Object
 The current row's data.
 
 ##### return: any
-The calculated value to be used for grouping.
+The calculated value to be used for grouping and group sorting.
+
+---
+
+The **DataGrid** groups its columns by the grouping column. If you want to use custom grouping, change values of this column in the **calculateGroupValue** function. As a result, group cells' headers will display the changed values that may include extra information. To remove it, specify a custom template in the [groupCellTemplate](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#groupCellTemplate) option ([groupCellRender](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#groupCellRender) in React). The example below uses this concept to display the *"California"* group always on top regardless of the sort order:
+
+---
+##### jQuery
```

Comment:
@pavel-2020 Just a note - we should replace this code with a link to github. The last sentence from the previous paragraph (The example below uses this concept to display the *"California"* group always on top regardless of the sort order:) should also be replaced with something like 'Review `[a project on GitHub](link)` that illustrates this scenario.'